src/test/java/com/jcabi/github/mock/MkCommentTest.java

```diff
@@ -90,14 +91,76 @@ public void canCompareInstances() throws Exception {
     }
 
     /**
+     * MkComment should store all its data properly.
+     * We should get the proper data back when accessing its properties.
+     * @throws Exception when a problem occurs.
+     */
+    @Test
+    public void dataStoredProperly() throws Exception {
+        final String cmt = "what's up?";
+        final long before = this.now();
+        final Comment comment = this.comment(cmt);
+        final long after = this.now();
+        MatcherAssert.assertThat(
+            comment.number(),
+            Matchers.greaterThan(0)
+        );
+        final Comment.Smart smart = new Comment.Smart(comment);
+        MatcherAssert.assertThat(
+            smart.issue().number(),
+            Matchers.greaterThan(0)
+        );
+        MatcherAssert.assertThat(
+            smart.author().login(),
+            Matchers.equalTo("jeff")
+        );
+        MatcherAssert.assertThat(
+            smart.body(),
+            Matchers.equalTo(cmt)
+        );
+        MatcherAssert.assertThat(
+            smart.url(),
+            // @checkstyle LineLength (1 line)
+            Matchers.equalTo(new URL("https://api.jcabi-github.invalid/repos/jeff/test/issues/comments/1"))
```

Comment:
@cvrebert Actually, it's better to format this line following way:
```java
		Matchers.equalTo(
				new URL(
						"https://api.jcabi-github.invalid/repos/jeff/test/issues/comments/1"
				)
		)
```
Sorry, we are very pedantic :)